components/automate-cli/cmd/chef-automate/pullAndGenerateConfig.go

```diff
@@ -1650,3 +1652,54 @@ func getGcsBackupConfig(a2ConfigMap map[string]*dc.AutomateConfig, fileUtils fil
 	}
 	return objStoage, nil
 }
+
+func (p *PullConfigsImpl) getBackupPathFromAutomateConfig(a2ConfigMap map[string]*dc.AutomateConfig, backupLocation string) (string, error) {
+	for _, ele := range a2ConfigMap {
+		path := ""
+		switch backupLocation {
+		case "fs":
+			if ele.Global.V1.External.Opensearch.Backup.Fs != nil &&
```

Comment:
can you handle here and below conditions if ele.Global.V1.External and ele.Global.V1.External.Opensearch and ele.Global.V1.External.Opensearch.Backup is not nil

Reply:
Done